workbench/scripts/watch.js

```diff
@@ -0,0 +1,20 @@
+const { createServer } = require('vite');
```

Comment:
I might have missed it, but is this just a helpful tool for the bundling service?

Reply:
This is the script that serves up the front-end code during development. It runs on `yarn run start`.